src/raytracer/Raytracer.m

```diff
@@ -130,7 +130,7 @@
     case 'LectureRoom'
             MaterialLibrary = importMaterialLibrary('raytracer/Material_library_LectureRoom.txt');
     case 'ParkingLot'
-            MaterialLibrary = importMaterialLibrary('raytracer/Material_library_ParkingLot.txt');
+            % MaterialLibrary = importMaterialLibrary('raytracer/Material_library_ParkingLot.txt');
```

Comment:
if the scenario still doesn't have a complete material library i would suggest to do one of the following possibilities:
* Remove the parking lot scenario
* Forcefully disable the QD switch in runtime showing a warning message and continue without QD

Reply:
We have updated the parameters. I will include this scenario now. Thanks

Reply:
should we wait for the update before merging the PR?

Reply:
Sorry at this moment, we cannot provide parameters. We should not wait for this. We can include these parameters in the future. Thanks

Reply:
We should delete parking lot at this moment